crates/flox/src/main.rs

```diff
@@ -17,7 +18,9 @@ async fn run(args: FloxArgs) -> Result<()> {
     init_logger(Some(args.verbosity.clone()), Some(args.debug));
     set_user()?;
     set_parent_process_id();
-    args.handle(config::Config::parse()?).await?;
+    let config = config::Config::parse()?;
+    init_global_manifest(&config.flox.config_dir.join("global-manifest.toml"))?;
```

Comment:
suggestion blocking: use `global_manifest_path()`

Reply:
Thinking more about this, the `global_manifest_path` function takes a `Flox` as an argument, and we don't have one at this point. So we can either construct the path ourselves here and keep the path abstracted away everywhere else, or we can have the same interface everywhere and pass in the config directory here. The point of the `global_manifest_path` function is to abstract away having to know anything about where the path is, so needing to pass the config directory in everywhere defeats the purpose. For that reason I think it's best to construct the path manually in this one spot.

Reply:
Ah I was thinking we had all we need because I saw `flox.config_dir`, but I'm fine leaving as is